components/Item/Item.js

```diff
@@ -1,15 +1,83 @@
 import styled from "styled-components";
+import { IoStar, IoLocationSharp } from "react-icons/io5";
 
 export default function Item({ item }) {
   return (
-    <StyledLi>
-      <h2>{item.title}</h2>
+    <StyledLi image={item.image}>
+      <StyledTitle>{item.title}</StyledTitle>
+      <StyledRating>
+        <IoStar />
+        {item.rating}
+      </StyledRating>
+      <StyledTags>
+        {item.tags.map((tag) => (
+          <StyledTag key={tag}>{tag}</StyledTag>
+        ))}
+      </StyledTags>
```

Comment:
```suggestion
      <StyledTagWrapper>
        {item.tags.map((tag) => (
          <StyledTag key={tag}>{tag}</StyledTag>
        ))}
      </StyledTagWrapper>
```
to differentiate better between the tag and the wrapper div